src/ui/descendents.rs

```diff
@@ -4,7 +4,10 @@ use std::fs::File;
 use std::io::Read;
 
 use failure::Error;
+#[cfg(unix)]
 use libc::pid_t;
+#[cfg(windows)]
+type pid_t = u32;
```

Comment:
could you add a `types` module at the top level that handles this cfg to avoid duplicating it?

Reply:
Done - but I used the core::types module instead of creating a new one